### ISSUE TITLE
deno install 好像无法安装了

### ISSUE THREAD

**Issue by keidarcy** [xcatliu/pagic#33]
**Describe the bug**
```
deno install --unstable --allow-read --allow-write --allow-net https://deno.land/x/pagic/mod.ts
```
无法安装。

**To Reproduce**
- 直接安装
```bash
❯ deno install --unstable --allow-read --allow-write --allow-net https://deno.land/x/pagic/mod.ts
Download https://deno.land/x/pagic/mod.ts
Warning Implicitly using latest version (v0.8.2) for https://deno.land/x/pagic/mod.ts
Download https://deno.land/x/pagic@v0.8.2/mod.ts
error: Import 'https://deno.land/x/pagic@v0.8.2/mod.ts' failed: 404 Not Found
```

- 手动安装之前版本
```bash
❯  deno install --unstable --allow-read --allow-write --allow-net https://deno.land/x/pagic@v0.8.1/mod.ts
.....
error: Import 'https://deno.land/x/types/react/v16.13.1/react.d.ts' failed: 404 Not Found
```

- 直接访问[0.82](https://deno.land/x/pagic@v0.8.2/mod.ts),也无法访问。

**Context**
- Pagic version: v0.8.2
- Deno version: deno 1.2.2
- Browser version (optional): 
- OS version: macOs 10.15.5

------
感觉很不错想试试看呀！

**Comment by hylerrix** [comment]
我也有类似错误，我暂时等 v1.0 发布再用吧 🤣

**Comment by lucacasonato** [comment]
Sorry this was a bug on deno.land/x. It should be fixed now. Don't forget to relink the repository with deno.land/x if you have not done so after the recent update: https://deno.land/posts/registry2

**Comment by xcatliu** [comment]
Fixed in v0.8.4
